pkg/controller/metrics/metrics.go

```diff
@@ -173,6 +199,7 @@ func Add(mgr manager.Manager) error {
 	}
 	metrics.Registry.MustRegister(newProvisioningUnderwaySecondsCollector(mgr.GetClient(), 1*time.Hour))
 	metrics.Registry.MustRegister(newProvisioningUnderwayInstallRestartsCollector(mgr.GetClient(), 1))
```

Comment:
Let's add a comment in here like
```suggestion
	// TODO: Make these optional & configurable via HiveConfig.Spec.MetricsConfig.
	metrics.Registry.MustRegister(newProvisioningUnderwaySecondsCollector(mgr.GetClient(), 1*time.Hour))
	metrics.Registry.MustRegister(newProvisioningUnderwayInstallRestartsCollector(mgr.GetClient(), 1))
```